src/polymer/behavior-scanner.ts

```diff
@@ -290,13 +290,24 @@ class BehaviorVisitor implements Visitor {
  * gets the expression representing a behavior from a node.
  */
 function behaviorExpression(node: babel.Node): babel.Node|null|undefined {
+  if (babel.isVariableDeclarator(node)) {
```

Comment:
So this function is only called when the node has been explicitly annotated with `@polymerBehavior`, right?

Reply:
Yes